### PR TITLE
[dom] Update Snabbdom to v3

### PR DESCRIPTION
**Pull request by ryota-ka**
Addresses #944

After this pull request is merged, Snabbdom v0.x will no longer be supported.
So a major version bump is required.

- [x] There exists an issue discussing the need for this PR
- [ ] I added new tests for the issue I fixed or built
- [x] I used `pnpm run commit` instead of `git commit`

### CODE REVIEW

**Comment by ryota-ka** [comment]
Probably this would also address #925.

File: dom/src/makeDOMDriver.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-import {Module} from 'snabbdom/modules/module';
+import {init, Module} from 'snabbdom';
 import xs, {Stream, Listener} from 'xstream';
 import concat from 'xstream/extra/concat';
 import sampleCombine from 'xstream/extra/sampleCombine';
 import {DOMSource} from './DOMSource';
 import {MainDOMSource} from './MainDOMSource';
-import {VNode} from 'snabbdom/vnode';
-import {toVNode} from 'snabbdom/tovnode';
+import {VNode,toVNode} from 'snabbdom';
```
Comment (jvanbruegge) [inline dom/src/makeDOMDriver.ts:8]: Could you move all the imports into a single statement?

**Review by jvanbruegge — COMMENTED** [review]
Looks good, basically just changing the imports. Exactly what I expected for the move to v3. That's nice to see.
Just one small nit pick, but LGTM

File: dom/src/makeDOMDriver.ts (hunk 1 above)
Comment (ryota-ka) [reply dom/src/makeDOMDriver.ts]: Merged into line 2! Thanks!

**Comment by jvanbruegge** [comment]
I've checked the tests myself and everything still works as expected.
@staltz if this looks good to you, can you merge and release?

**Comment by staltz** [comment]
Yes, l'll do that

**Pull request merged**